app/src/main/java/org/wikipedia/readinglist/ReadingListBookmarkMenu.java

```diff
@@ -73,7 +77,7 @@ private void showMenu() {
         menu.getMenuInflater().inflate(menuRes, menu.getMenu());
         menu.setOnMenuItemClickListener(new PageSaveMenuClickListener());
 
-        if (listsContainingPage.size() == 1) {
+        if (size(listsContainingPage) == 1) {
```

Comment:
Why do we need a third-party library to get the size of a list?

Reply:
That particular list is nullable, so calling its size() method directly might throw a `NullPointerException`.